app/src/main/res/layout/mainactivity/drawable/pink_out_line.xml

```diff
@@ -0,0 +1,8 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I would rename the file in green_outline or something similar, just to make it coherent with the content.

Reply:
Ahh, I did not notice the strange file name, will change it